### ISSUE TITLE
cleandb

### ISSUE THREAD

**Issue by 532910** [roundcube/roundcubemail-docker#110]
Roundcube provides https://github.com/roundcube/roundcubemail/blob/master/bin/cleandb.sh which debian runs by cron. Should it added here?

**Comment by sebastianhelbig** [comment]
I also asked myself this when switching to the apache-based docker image. There is no mention of this other than in the kubernetes example. Is cleandb still needed?

**Comment by pabzm** [comment]
Yes, it would be good to integrate such a task. But it needs some consideration, as our container image runs apache/php-fpm as init process (as is proper), so there's no cron daemon.

Anyone up for it?
